Source/Mosa.DeviceDriver/ScanCodeMap/US.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
 
 using Mosa.DeviceSystem;
+using Mosa.DeviceSystem.Devices.Keyboard;
+using Mosa.DeviceSystem.Drivers.Keyboard;
```

Comment:
Duplicate

Reply:
one is Devices.Keyboard while the other is Drivers.Keyboard